src/browser/starter.js

```diff
@@ -1029,7 +1029,10 @@ V86Starter.prototype.screen_go_fullscreen = function()
         var focus_element = document.getElementsByClassName("phone_keyboard")[0];
         focus_element && focus_element.focus();
     }
-
+    // Use keyboard lock when possible
+    if(navigator?.keyboard?.lock instanceof Function) {
+        try {navigator.keyboard.lock();}catch(e) {};
+    }
```

Comment:
`?` isn't supported by our build tools, how about:

```
try {
   navigator.keyboard.lock();
}
catch(e) {}
```

Reply:
@copy Okay, I thought I should avoid that, because it can fail for other reasons than a lack of support.

Reply:
Resolved in c66ad25fb9b825d30b54ce7b1c66e40f7528d67b